test/lib/components/CopyButton.spec.ts

```diff
@@ -2,6 +2,10 @@ import { fireEvent, render, waitFor } from '@testing-library/svelte';
 
 import CopyButton from '$lib/components/util/CopyButton.svelte';
 
+function getFirstNonCommentChild(node: Node) {
```

Comment:
Svelte now uses comments for hydration, so the first children were comments, which broke the tests. https://svelte.dev/docs/svelte/v5-migration-guide